jobrunner/jobrunner.service

```diff
@@ -9,7 +9,23 @@ EnvironmentFile=/srv/jobrunner/environ/02_secrets.env
 EnvironmentFile=/srv/jobrunner/environ/03_backend.env
 EnvironmentFile=/srv/jobrunner/environ/04_local.env
 WorkingDirectory=/srv/jobrunner/code
-#ExecStartPre=TODO
+
+# Creates and configures a slice which allows network access to only the IPs
+# listed in DATABASE_IP_LIST. Any Docker container run with:
+#
+#     --cgroup-parent=$DATABASE_ACCESS_SLICE
+#
+# will be subject to this restriction.
+Environment="DATABASE_IP_LIST="
```

Comment:
Am assuming this is a default value, and EnvironmentFile will override?

Maybe worth a comment to clarify if so?

Reply:
Yes, good call